tests/yam/yast_modules/control_center.pm

```diff
@@ -10,19 +10,27 @@
 use base 'y2_installbase';
 use strict;
 use warnings;
-use testapi qw(select_console enter_cmd wait_still_screen save_screenshot get_var);
+use testapi;
 
 sub run {
     my $podman_cmd = "podman container runlabel run registry.opensuse.org/suse/alp/workloads/tumbleweed_containerfiles/suse/alp/workloads/yast-mgmt-ncurses-test-api";
     select_console('root-console');
-    enter_cmd(get_var('YUI_PARAMS') . " $podman_cmd");
+    my $podman_start = get_var('YUI_PARAMS') . " ZYPP_MEDIA_CURL_DEBUG=1 Y2STRICTTEXTDOMAIN=1 $podman_cmd;";
+    if (!script_run($podman_start . " echo podman-status-\$? > /dev/$serialdev", 0)) {
+        record_info("Podman start: ", $podman_start);
```

Comment:
this code looks like a copy/paste from  existing functions `with_yast_env_variables`  and `yast2_console_exec`, why not to  switch the yast2 command by the podman one in ? https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/6ba0c168f14fa63cc1d471521a7bc24cf7b9b244/lib/y2_module_consoletest.pm#L15
instead of duplicating functionality?

Reply:
Done

Reply:
you are not modifying function `yast2_console_exec` so I don't think you considered or get my suggestion.